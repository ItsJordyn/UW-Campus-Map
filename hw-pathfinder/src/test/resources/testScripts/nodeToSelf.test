# Find a path with Dijkstra's algorithm when
# the path is from a node to itself

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C

AddEdge simpleGraph A B 9.0
AddEdge simpleGraph B C 2.0

FindPath simpleGraph A A