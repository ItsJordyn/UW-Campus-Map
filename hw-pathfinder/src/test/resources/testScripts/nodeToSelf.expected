# Find a path with Dijkstra's algorithm when
# the path is from a node to itself

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph

added edge 9.000 from A to B in simpleGraph
added edge 2.000 from B to C in simpleGraph

path from A to A:
total cost: 0.000